src/main/java/io/orangebeard/listener/OrangebeardTestSystemListener.java

```diff
@@ -358,12 +372,15 @@ private static String getFitnesseRootPath(String propertyFileName) {
         return fitnesseRootPath;
     }
 
-    private StartTestItem getStartTestItem(TestPage testPage) {
-        StartTestItem.StartTestItemBuilder testItem = StartTestItem.builder()
+    private StartTest getStartTestItem(TestPage testPage) {
+         String fullSuiteName = TestPageHelper.getFullSuiteName(testPage);
+
+        StartTest.StartTestBuilder testItem = StartTest.builder()
                 .testRunUUID(runContext.getTestRunUUID())
-                .startTime(LocalDateTime.now())
-                .name(getTestName(testPage))
-                .type(determinePageType(testPage.getName()));
+                .startTime(ZonedDateTime.now())
+                .testName(getTestName(testPage))
+                .testType(TestType.valueOf(NewUIdeterminePageType(testPage.getName()).toString()))
```

Comment:
- NewUIDeterminePageType? It has nothing to do with the new ui, it's about the v3 api. 
- Why are you doing a tostring and then a value of? 
- Methods should not have a capital letter